src/focus-ring.js

```diff
@@ -74,11 +80,13 @@ document.addEventListener('DOMContentLoaded', function() {
    * opening a menu or dialog.
    */
   function onKeyDown() {
-    hadKeyboardEvent = true;
     // `activeElement` defaults to document.body if nothing focused,
     // so check the active element is actually focused.
-    if (matches(document.activeElement, ':focus'))
-      addFocusRingClass(document.activeElement);
+    var activeElement = document.activeElement;
+    if (activeElement.tagName == 'BODY')
+      return;
+
+    hadKeyboardEvent = true;
```

Comment:
`hasKeyboardEvent` is not intended for that use, though. I think this could cause a bug if a keyboard event triggered a focus move (for example, activating a button which caused a piece of modal UI to appear). Please revert this change.